src/ui/badge.svelte

```diff
@@ -27,7 +27,7 @@
     text-transform: uppercase;
   }
 
-  @media screen and (min-width: 768px) {
+  @container layout-query-container (width > 768px) {
```

Comment:
93% support, which is very acceptable for our use case I think. We could in the future provide a fallback however